scripts/hyperkit.mjs

```diff
@@ -5,50 +5,111 @@ import os from 'os';
 import process from 'process';
 import childProcess from 'child_process';
 import path from 'path';
+import util from 'util';
 
 // The version of hyperkit to build
 const ver = 'v0.20210107';
 
+// Command lines for sudo.  Note that this will be passed to `sh -c`.
+const sudoTasks = [];
+
 /**
  * Build the Hyperkit binary.
- * @param workPath {string} The directory to work in; must be empty.
+ * @param destPath {string} The output path for the binary.
  * @returns {string} The executable in the work directory.
  */
-async function buildHyperkit(workPath) {
+async function buildHyperkit(destPath) {
```

Comment:
I was initially confused that `workPath` points to a directory and `destPath` points to a file. I guess there is nothing in `Path` that should make me assume it is a directory, but somehow it does. So I might find `workDir` and `destFile` easier to grasp intuitively, but don't feel strongly enough to make you change it...